.github/workflows/ci.yml

```diff
@@ -291,7 +291,22 @@ jobs:
         run: |
           ${{ matrix.cmd }}
   
-  # ──────────────────────── 5 · Rest catalog test (feature) ─────────────────
+  # ──────────────────────── 5 · Otel test (feature) ─────────────────
+  otel_test:
+    name: Opentelemetry tests (feature)
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - uses: actions/checkout@v5
```

Comment:
I mean, we should keep code consistency for all workflow setups.
https://github.com/Mooncake-Labs/moonlink/blob/9346a832d239110d10777e61e996477684b28d1f/.github/workflows/ci.yml#L351-L366
Take ^ as an example, what's the benefit for your implementation over existing ones?
As I mentioned [here](https://github.com/Mooncake-Labs/moonlink/pull/2141/files#r2390490607)